packages/components/src/Autocomplete/Menu.tsx

```diff
@@ -44,7 +47,17 @@ export function Menu({
   useEffect(() => setHighlightedIndex(initialHighlight), [options]);
 
   return (
-    <div className={optionMenuClass}>
+    <div
+      className={optionMenuClass}
+      style={{ scrollBehavior: "smooth" }}
```

Comment:
Can you add this to the `optionMenuClass` instead of adding a style attribute?

Reply:
Yup, added the style into the css file!